### DISCUSSION TITLE
Locked Out of GitHub Company Accounts - Urgent Assistance Needed

### DISCUSSION THREAD

**Discussion by Grace-MT** [community/community#142807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

My colleague and I have both been unexpectedly locked out of our company’s GitHub accounts without any explanation or prior notice. We haven't engaged in any suspicious activity and are confident that we've adhered to GitHub's terms of service. As a small company, access to our projects is crucial, especially with a new team member joining soon who will need access.

I submitted a support ticket to GitHub on Monday, October 21st, and received a confirmation email, but we have yet to hear back. I’ve attached a screenshot of the error message we’re seeing for reference.

We’re really hoping for a swift resolution to regain access. If anyone has advice or has experienced something similar, your help would be greatly appreciated!

Thank you!
<img width="259" alt="Screenshot 2024-10-28 092803" src="https://github.com/user-attachments/assets/9bf8fc69-7b39-40bd-a4b0-2a1e714eec2d">

**Comment by MasterInQuestion** [comment]
My recent research indicated GitHub Support may take months to respond... or maybe never respond.
    https://github.com/orgs/community/discussions/43929
    https://github.com/orgs/community/discussions/134510
    https://github.com/orgs/community/discussions/135697
    https://github.com/orgs/community/discussions/142305

    https://github.com/orgs/community/discussions/134410#discussioncomment-10328752
    Never trust remote storage for critical data.

**Comment by LiteBrite82** [comment]
Hi @Grace-MT 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
